packages/neuron-ui/src/components/History/index.tsx

```diff
@@ -181,64 +181,49 @@ const History = () => {
-          if (item.nftInfo) {
-            // NFT
-            const { type, data } = item.nftInfo
-            amount = show ? `${type === 'receive' ? '+' : '-'}${nftFormatter(data)}` : `${HIDE_BALANCE}mNFT`
-            isReceive = type === 'receive'
-          } else if (item.sudtInfo?.sUDT) {
-            if (item.sudtInfo.sUDT.decimal) {
-              sudtAmount = sUDTAmountFormatter(sudtValueToAmount(item.sudtInfo.amount, item.sudtInfo.sUDT.decimal))
-            }
-          } else {
-            amount = show ? `${shannonToCKBFormatter(item.value, true)} CKB` : `${HIDE_BALANCE} CKB`
-            isReceive = !amount.includes('-')
-          }
-        }
-
-        return (
-          <CopyZone content={`${complexNumberToPureNumber(sudtAmount && sudtAmount !== '' ? sudtAmount : amount)}`}>
-            {sudtAmount ? (
-              <>
-                {show ? sudtAmount : HIDE_BALANCE}&nbsp;
-                <UANTonkenSymbol
-                  className={styles.symbol}
-                  name={item.sudtInfo!.sUDT.tokenName}
-                  symbol={item.sudtInfo!.sUDT.symbol}
-                />
-              </>
-            ) : (
-              <span className={show && isReceive ? styles.isReceive : ''}>{amount}</span>
-            )}
-          </CopyZone>
-        )
+        return <FormattedTokenAmount item={item} show={show} isNeedCopy />
       },
     },
     {
       title: t('history.table.timestamp'),
       dataIndex: 'timestamp',
       align: 'left',
+      minWidth: '150px',
       render: (_, __, item) => uniformTimeFormatter(item.timestamp),
     },
     {
       title: t('history.table.status'),
       dataIndex: 'status',
       align: 'left',
+      minWidth: '50px',
       render(_, __, item) {
-        const confirmations = 1 + bestBlockNumber - +item.blockNumber
-        let status = item.status as string
-        if (status === 'success' && confirmations < CONFIRMATION_THRESHOLD) {
+        const confirmationNum = 1 + bestBlockNumber - +item.blockNumber
+        let status = item.status as TransactionStatusWrapProps['status']
+        let confirmationCount = ''
+
+        if (status === 'success' && confirmationNum < CONFIRMATION_THRESHOLD) {
           status = 'confirming'
+
+          if (confirmationNum === 1) {
+            confirmationCount = t('history.confirmation', {
+              confirmationCount: localNumberFormatter(confirmationNum),
+            })
+          } else if (confirmationNum > 1) {
+            confirmationCount = `${t('history.confirmations', {
+              confirmationCount: localNumberFormatter(confirmationNum),
+            })}`
+          }
         }
 
-        return status === 'confirming' ? <Confirming className={styles.confirm} /> : t(`history.${status}`)
+        return (
+          <TransactionStatusWrap
+            status={status}
+            statusText={t(`history.${status}`)}
+            confirmationCount={confirmationCount}
```

Comment:
Could we only transfer `status` and `confirmationCount` into `TransactionStatusWrap` component without i18n information?

Reply:
Sounds good.